lib/__tests__/geocodioLibraryNode.test.js

```diff
@@ -200,4 +200,35 @@ describe("geocoder", () => {
       expect(response.results.length).toEqual(1);
     });
   });
+
+  // LIST TESTS
+
+  it("Can create a new list, check status, download list and delete list", () => {
+    expect.assertions(1);
+
+    let newList = geocoder.list
+      .create(
+        `${__dirname}/stubs/sample_list.csv`,
+        "forward",
+        "{{A}} {{B}} {{C}} {{D}}",
+        "https://example.com/my-callback"
+      )
+      .then(response => {
+        expect(response.file.filename).toEqual("sample_list.csv");
+
+        geocoder.list.status(response.id).then(resp => {
+          expect(resp.id).toEqual(response.id);
+
+          geocoder.list.download(response.id, "newList").then(resp => {
+            expect(resp.file.filename).toEqual("newList");
```

Comment:
I think the `download()` method should take a filename to save to, i.e. `geocoder.list.download(response.id, "geocoded_file.csv")`. The response should also be a CSV file rather than JSON (once the file has been fully processed)